[ENHANCEMENT] Add `mimir-http-prefix` option to introduce a prefix for Mimir URL when use legacy routes. (#8069)
grafana/mimir: CHANGELOG.md

* [CHANGE] Deprecated `--rule-files` flag in favor of CLI arguments. #7756
* [BUGFIX] Fix panic in `loadgen` subcommand. #7629

> * [ENHANCEMENT] Add `mimir-http-prefix` option to introduce a prefix for Mimir URL when use legacy routes. #8069

Reviewer: Nit
```suggestion
* [ENHANCEMENT] Add `mimir-http-prefix` configuration to set the Mimir URL prefix when using legacy routes. #8069
```
